src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+      class="big"
```

Comment:
Due to the checklist you can create general class for example big-form and move all similar styles under it and add another additional class-modifier to your small form and other elements
![image](https://user-images.githubusercontent.com/80700247/229532206-349e68a0-b2eb-4b2c-a9e5-77f382b84ce4.png)